arm-logic/2016-06-01/examples/SessionsCreateOrUpdate.json

```diff
@@ -0,0 +1,40 @@
+{
+  "title": "Create or update integration account session example",
+  "parameters": {
+    "api-version": "2016-06-01",
+    "subscriptionId": "34adfa4f-cedf-4dc0-ba29-b6d1a69ab345",
+    "resourceGroupName": "testrg123",
+    "integrationAccountName": "testia123",
+    "sessionName": "testsession123-ICN",
+    "session": {
+      "properties": {
+        "content": "{\"controlNumber\":\"1234\",\"controlNumberChangedTime\":\"2017-02-21T22:30:11.9923759Z\"}"
```

Comment:
IntegrationAccountSession is defined as "allOf" Resource, generally "location" property is a "required" property, in which it may need to be included here. In the definition of Resource included in logic.json I don't see it marked as required, so you may just need to check on this. If you test the API with this set of parameters, does it return successfully?